careerfairsystems/arkad-app
# Connect remove student backend

## Change description

The user now gets correct msg if wrong password or username. The user will first be logged in and then it will start to load the content depending if it is a student or company. 

Hopefully the remove student will work now. 

## How to verify

Login as a company and see if it works. Login with wrong credentials and see if correct info shows. 

We can check if remove student works when we meet. 

## Issues fixed

None
